aspnetcore/fundamentals/middleware/index.md

```diff
@@ -156,6 +156,12 @@ public void Configure(IApplicationBuilder app)
 }
 ```
 
+For SPA app's, see the code generated by any of the following templates:
+
+* `dotnet new react`
+* `dotnet new angular`
+* `dotnet new react`
```

Comment:
- React is twice here.
- Plural of app is “apps” not “app's”. The A in SPA is actually already application, so the app is double.
- Instead of referring to the dotnet commands, maybe we can just link to the SPA docs on [Angular](https://docs.microsoft.com/en-us/aspnet/core/client-side/spa/angular?view=aspnetcore-3.1) and [React](https://docs.microsoft.com/en-us/aspnet/core/client-side/spa/react?view=aspnetcore-3.1) instead to allow people to get some more information about this, and not to lock them into the command line experience (VS has these templates too or not?)
- I actually also feel that this mentioning of SPA is a bit out of place here. Maybe we can expand this a bit to make this more substantial. Something like:

> For single page applications, the SPA middleware (`UseSPA`) will usually come last in the middleware pipeline to respond to all other requests. This allows all other middlewares to respond to matching requests first, while allowing SPAs with client-side routing to run for all routes that are unrecognized by the server application. For more details on single page applications, see the guides for the [React](https://docs.microsoft.com/en-us/aspnet/core/client-side/spa/react?view=aspnetcore-3.1) and [Angular](https://docs.microsoft.com/en-us/aspnet/core/client-side/spa/angular?view=aspnetcore-3.1) project templates.